File: PAV 1-TPI - CASOS DE USO version 1/Resueltos/Usuarios+SeleccionActBugs/App_BugTracker resuelto/App_BugTracker/BusinessLayer/Services/CriticidadService.vb
Public Class CriticidadService
    Private oCriticidadDao As CriticidadDao
    Public Sub New()
        oCriticidadDao = New CriticidadDao
    End Sub

    Public Function consultarCriticidades() As List(Of Criticidad)
        Return oCriticidadDao.getAll
    End Function
End Class

File: Clases 2018/Clases-C#/ClasePractica12_2018 (transacciones con data manager)/C#_TPS_BugTracker/TPS_BugTracker/BusinessLayer/Services/CriticidadService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TPS_BugTracker.BusinessLayer.Entities;
using TPS_BugTracker.DataLayer.DAOs;

namespace TPS_BugTracker.BusinessLayer.Services
{
    class CriticidadService
    {
        private CriticidadDao oCriticidadDao;
        public CriticidadService()
        {
            oCriticidadDao = new CriticidadDao();
        }

        public List<Criticidad> consultarCriticidades()
        {
            return oCriticidadDao.getAll();
        }
    }
}
